Braintree.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,14 @@
+{
+  "pins" : [
+    {
+      "identity" : "paypalcheckout-ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/paypal/paypalcheckout-ios/",
+      "state" : {
+        "revision" : "7c6750e1316c6a3d656e90497271de68c63219f1",
+        "version" : "1.1.0"
+      }
+    }
+  ],
+  "version" : 2
+}
```

Comment:
This file shouldn't be added (we can add it to the gitignore in another PR)

Reply:
ok I'll remove it